src/main/java/com/fasterxml/jackson/core/io/IOContext.java

```diff
@@ -141,6 +150,30 @@ public IOContext(StreamReadConstraints src, StreamWriteConstraints swc, BufferRe
         _contentReference = contentRef;
         _sourceRef = contentRef.getRawContent();
         _managedResource = managedResource;
+        _errorReportConfiguration = ErrorReportConfiguration.defaults();
```

Comment:
Maybe instead call:

    this(src, swc, br, contentRef, managedResource, ErrorReportConfiguration.defaults();